rocminfo.cc

```diff
@@ -997,6 +1009,44 @@ AcquireAndDisplayAgentInfo(hsa_agent_t agent, void* data) {
   return HSA_STATUS_SUCCESS;
 }
 
+void CheckInitError(hsa_status_t err) {
+
+  printf("ROCm initialization failed: %d\n", err);
```

Comment:
I was going to change this to be hex to make it easier to look up in the hsa documentation/header. We can do it in a later commit if you prefer.

Reply:
You can translate ROCr error codes to strings with hsa_status_string.  That might be more human friendly.

Reply:
I was going to make the change to use hsa_status_string(), but found that it won't work for when hsa_init() fails. I believe the IS_OPEN() check at the beginning of hsa_status_string() fails. But I should add this for the other hsa_* failures.

Reply:
That's probably a bug in ROCr (or in the HSA spec and we should fix it).  Translating error codes shouldn't require hsa_init to be open since it's static info.